PowerShellWorker.Common.props

```diff
@@ -6,5 +6,6 @@
     <PackageLicenseUrl>https://github.com/Azure/azure-functions-powershell-worker/blob/dev/LICENSE</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/Azure/azure-functions-powershell-worker</RepositoryUrl>
+    <UseRidGraph>Alternate</UseRidGraph>
```

Comment:
Could you please include a comment in the PR description explaining why we need this for `PowerShell 7.2`?

Reply:
This change was actually unintentional - reverted